Automattic/jetpack
# Untangle: Move Settings > Newsletter to Jetpack

Fixes https://github.com/Automattic/dotcom-forge/issues/5461

## Proposed changes:
Move Newsletter submenu from Settings to Jetpack, and show it under `Settings` or `Subscribers` (if https://github.com/Automattic/jetpack/pull/35429 is merged), and add an external link icon.

| Before | After |
| --- | --- |
| ![image](https://github.com/Automattic/jetpack/assets/402286/87cc4cd1-98f0-4533-9b35-ffdfc2ba92aa) | ![image](https://github.com/Automattic/jetpack/assets/402286/c83c94ad-c20b-48e0-82c7-b113a8382814) |

## Jetpack product discussion
Details on Issue: https://github.com/Automattic/dotcom-forge/issues/5445

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
* Set your `Admin interface style` to `Classic (wp-admin)` in `wordpress.com/hosting-config/site_slug`
* Go to `wp-admin`
* The Jetpack menu should have the Newsletter item with an external link icon.
* The Settings menu should not have the Newsletter item.
